charts/posit-chronicle/values.yaml

```diff
@@ -0,0 +1,85 @@
+image:
+  repository: "935931255537.dkr.ecr.us-east-2.amazonaws.com/chronicle"
+  tag: "latest"
+  imagePullPolicy: "Always"
+
+serviceaccount:
+  enabled: false
+  # -- Additional annotations to add to the chronicle-server serviceaccount
+  annotations: {
+    # EKS role arn example
+    # eks.amazonaws.com/role-arn: my-aws-iam-role-arn
+  }
+  # -- Additional labels to add to the chronicle-server serviceaccount
+  labels: {}
+
+service:
+  rest:
+    # -- The port to use for the REST service
+    port: 80
+    # -- The port to forward REST requests to on the pod. Also see pod.port
+    targetPort: 5252
+    # -- Additional annotations to add to the chronicle-server service
+    annotations: {}
+    # -- Additional labels to add to the chronicle-server service
+    labels: {}
+
+# -- The number of replica pods to maintain for this service
+replicas: 1
+
+pod:
+  command: "/chronicle"
+  args: ["start", "-c", "/opt/chronicle/config.gcfg"]
+  rest:
+    port: 5252
+  # -- Optional environment variables
+  env: []
+  # -- Additional annotations to add to the chronicle-server pods
+  annotations: {}
+  # -- Additional labels to add to the chronicle-server pods
+  labels: {}
+  # -- A map used verbatim as the pod's "affinity" definition
+  affinity: {}
+  # -- A map used verbatim as the pod's "nodeSelector" definition
+  nodeSelector: {}
+  # -- An array used verbatim as the pod's "tolerations" definition
+  tolerations: []
+
+# We do want at least 1Gi of persistent volume space
+# to perform compaction/compression operations
+storage:
+  persistentVolumeSize: 1Gi
+
+config:
+  http:
+    listen: ":5252"
+  https:
+    # If https.enabled=true, we will ignore any http
+    # values and enable https in our config instead
+    enabled: false
+    listen: ":443"
+    key: ""
+    certificate: ""
+  localStorage:
+    enabled: true
+    location: "./chronicle-data"
```

Comment:
I don't see a reason to allow admins to set the mount path inside the container, so I think we should remove this option and have the generated config/StatefulSet default to something like `/data`.

```suggestion
```

Reply:
I do kindof like the extra documentation of having this at the top level of our helm values, volume mappings can be a little tricky to grok sometimes just from the code.
It's not needed though I agree

Reply:
Also we'll reuse this value in a couple places, it's also nice to just be able to reference `config.localStorage.location` when we do. That argument could have been valid for our port definitions we've since hardcoded too, though i'm fine with setting those directly